REFERENCE.md

```
@@ -683,6 +687,13 @@ f[x |-> 0, y |-> 1]     // Multiple overrides
 all k: Key, v: Value | mapping[k |-> v] k = v.
 ```
 
+For an N-ary rule, the override key is a tuple whose arity matches the rule's parameter list:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)